article/app/views/fragments/articleHeaderCommentGarnett.scala.html

```diff
@@ -67,21 +62,11 @@ <h1 class="content__headline" articleprop="headline" @langAttributes(article.con
                     <meta articleprop="name" content=".@* we can't know *@">
                 </div>
             }
-
-            @if(showBadge && !isPaidContent(page)) {
-                @fragments.commercial.badge(article, page)
-            }
-
```

Comment:
Shouldn't we keep the commercial badge (if `showBadge` is `true`) on a non-paid content article header?

Reply:
Hmm, I saw it [there](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/contentMeta.scala.html#L98) and in my tests it worked well, but perhaps there are examples I missed 🤔

Reply:
I also removed it because everywhere it was called, `showBadge = isPaidContent` and thus `showBadge && !isPaidContent` was always false